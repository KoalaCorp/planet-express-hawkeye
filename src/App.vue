<template>
  <div>
    <router-link to="/">
      Home
    </router-link> | <router-link to="/visualization">
      Visualization
    </router-link>
    <router-view />
  </div>
</template>
