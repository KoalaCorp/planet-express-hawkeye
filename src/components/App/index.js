import React from "react"
import { Switch, Route } from "react-router-dom"

import Home from "../../pages/Home"
import AboutProject from "../../pages/AboutProject"
import AboutUs from "../../pages/AboutUs"
import Visualization from "../../pages/Visualization"
import NoMatch from "../../pages/NoMatch"

const App = () => {
  return (
    <React.Fragment>
      <Switch>
        <Route exact path="/" component={Home} />
        <Route exact path="/about" component={AboutProject} />
        <Route exact path="/team" component={AboutUs} />
        <Route exact path="/visualization" component={Visualization} />
        <Route component={NoMatch} />
      </Switch>
    </React.Fragment>
  )
}

export default App
