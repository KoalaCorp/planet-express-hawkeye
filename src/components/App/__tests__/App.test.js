import React from "react"
import { render } from "../../../../tests/helpers"

import App from "../"

const renderComponent = ({ ...props } = {}) => render(<App {...props} />)

describe("Application", () => {
  it("renders correctly", () => {
    const app = renderComponent()
    expect(app).toMatchSnapshot()
  })
})
