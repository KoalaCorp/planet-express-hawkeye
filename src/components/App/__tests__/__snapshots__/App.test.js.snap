// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="header"
      >
        <h1
          class="header__title"
        >
          Planet Express
        </h1>
        <h2
          class="header__subtitle"
        >
          The auto relational searcher for political documents in Spain
        </h2>
      </header>
      <main>
        <div>
          Home
        </div>
      </main>
      <footer
        class="footer"
      >
        <ul
          class="footer__navigation"
        >
          <li>
            <a
              href="/about"
            >
              about the project
            </a>
          </li>
          <li>
            <a
              href="/team"
            >
              about the team
            </a>
          </li>
        </ul>
        <div
          class="footer__license"
        >
          this project is under a
           
          <a
            href="https://www.gnu.org/licenses/gpl-3.0.en.html"
            rel="license external noopener noreferrer"
            title="GNU General Public License v3.0"
          >
            GNU General Public License v3.0
          </a>
           
          license
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <header
      class="header"
    >
      <h1
        class="header__title"
      >
        Planet Express
      </h1>
      <h2
        class="header__subtitle"
      >
        The auto relational searcher for political documents in Spain
      </h2>
    </header>
    <main>
      <div>
        Home
      </div>
    </main>
    <footer
      class="footer"
    >
      <ul
        class="footer__navigation"
      >
        <li>
          <a
            href="/about"
          >
            about the project
          </a>
        </li>
        <li>
          <a
            href="/team"
          >
            about the team
          </a>
        </li>
      </ul>
      <div
        class="footer__license"
      >
        this project is under a
         
        <a
          href="https://www.gnu.org/licenses/gpl-3.0.en.html"
          rel="license external noopener noreferrer"
          title="GNU General Public License v3.0"
        >
          GNU General Public License v3.0
        </a>
         
        license
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
